building/target_platforms/target_platforms_for_consumers.adoc

```diff
@@ -0,0 +1,212 @@
+= Using Target Platforms
+
+This document explains how to consume target platforms in Eclipse.
+
+== What is a target platform?
+
+A target platform, in abstract terms, is a set of external dependencies that your features, plugins, and tests will use and on which their development and installation will depend. For JBoss Tools, these dependencies come from various Eclipse.org projects and other providers. Our target platforms can be used inside Eclipse to:
+
+* provide the right set of dependencies with which to debug, test, and develop;
+* provide the right set of dependencies at build time; and
+* provide a site containing all dependencies to end-users at install time. 
+
+In concrete terms, a target platform can take two forms: a `target platform definition file` and a `target platform update site`. 
+
+A `target platform definition file` is a custom Eclipse file, with a `.target` suffix, for use by Eclipse's Plug-in Development Environment (PDE) tools. The file lists  the versions of installable units (IUs) to be used as dependencies, as well as the repositories from which those dependencies can be found. The contained IUs are generally features, but can also be invididual plugins.
+
+A `target platform update site` is the result of `resolving` the target platform definition file, which is the process of downloading all the IUs, and thier contained IUs, to one's local disk. It includes the full set of jars for all dependencies our project may depend on, as well as the required Eclipse metadata to act as an update site. An update site, in Eclipse, is a mechanism used to install new (or update existing) software into Eclipse. Because of this, you could probably install the entire target platform into your Eclipse using such an update site. 
+
+See these slides for more about target platforms: http://www.slideshare.net/mickaelistria/a-journey-with-target-platforms.
+
+== Where is the target platform for JBoss Tools?
+
+JBoss Tools' `target platform definition files` are stored in github:
+
+* https://github.com/jbosstools/jbosstools-target-platforms[JBoss Tools & Red Hat JBoss Developer Studio Target Platforms]
+* https://github.com/jbosstools/jbosstools-discovery[JBoss Tools Central Target Platforms]
+* https://github.com/jbosstools/jbosstools-integration-stack[Integration Stack Target Platforms]
+
+JBoss Tools' `target platform update sites` are located here, in both unpacked and packed (zipped) format. They do *not* contain sources or SDK features.
+
+* http://download.jboss.org/jbosstools/targetplatforms/ (SNAPSHOTS and milestones)
+* http://download.jboss.org/jbosstools/static/targetplatforms/ (released versions)
```

Comment:
This is not about documenting *actual* content, but about documenting what urls are the ones to use/rely on.

/static/ is an implementation detail - the urls to use should not contain /static/ *anywhere*. The only place static should be mentioned is in where document how we upload the content imo.